### ISSUE TITLE
add index annotator for map query

### ISSUE THREAD

**Issue by toastjpg** [JoshuaJLi/cmpt362-wtb#69]
For map view, since we query for stops on the lat and lon fields, should add the @Index annotator to the data models.

Will increase realm db size, but will improve query speeds. 

Do this after #56 and #68 are merged into main. Should be the last change to the db file in assets.
